.bumpversion.cfg

```diff
@@ -1,9 +1,18 @@
 [bumpversion]
 commit = False
 tag = False
-current_version = 0.1.0
+current_version = 1.0.0-rc.0
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-(?P<pre>[a-z]+)\.(?P<build>\d+))?
+serialize = 
+	{major}.{minor}.{patch}-{pre}.{build}
+	{major}.{minor}.{patch}
```

Comment:
There are some limitations with this, but I wasn't immediately able to reproduce that in my local env. See https://github.com/c4urself/bump2version/issues/190